app/assets/javascripts/admin/addon/controllers/admin.js

```diff
@@ -1,11 +1,14 @@
 import Controller from "@ember/controller";
+import { alias } from "@ember/object/computed";
 import { inject as service } from "@ember/service";
 import { dasherize } from "@ember/string";
 import discourseComputed from "discourse-common/utils/decorators";
 
 export default class AdminController extends Controller {
   @service router;
 
+  @alias("siteSettings.enable_admin_sidebar_navigation") showAdminSidebar;
```

Comment:
So remember alias in two ways:

- you get the value
- but if you do this.showAdinSidebar = "xx"; then it mutates `this.siteSettings.enable_admin_sidebar_navigation`

I do you want this here. A simple getter should do what you want.

Reply:
yes should be fine